pangeo_forge_recipes/rechunking.py

```diff
@@ -238,3 +239,42 @@ def _sort_by_speed_of_varying(item):
     ds_combined = xr.combine_nested(dsets_to_concat, concat_dim=concat_dims_sorted)
 
     return first_index, ds_combined
+
+
+def _gather_coordinate_dimensions(group: zarr.Group) -> List[str]:
+    return list(
+        set(itertools.chain(*(group[var].attrs.get("_ARRAY_DIMENSIONS", []) for var in group)))
```

Comment:
I imagine since this is a `zarr.Group` this iteration isn't very expensive b/c it's on summary data and never really that large?

Reply:
Ehh, I think you're right? Since it's just scanning across vars in groups and looking at the .attrs.